src/Entity/Contractor.php

```diff
@@ -308,4 +320,43 @@ public function setSettings(ContractorSettings $settings): self
 
         return $this;
     }
+
+    /**
+     * @return Collection|Review[]
+     */
+    public function getReviews(): Collection
+    {
+        return $this->reviews;
+    }
+
+    /**
+     * @param Review $review
+     * @return $this
+     */
+    public function addReview(Review $review): self
+    {
+        if (!$this->reviews->contains($review)) {
+            $this->reviews[] = $review;
```

Comment:
```suggestion
            $this->reviews->add($review);
```
Arraycollection add method is more readable :)